lib/menu.rb

```diff
@@ -0,0 +1,24 @@
+class Menu
+  attr_reader :dishes
+
+  def initialize(dishes = default_dishes)
+    @dishes = dishes
+  end
+
+  def default_dishes
+    dishes = []
+    dishes << Dish.new("Curry", 8)
+    dishes << Dish.new("Rice", 3)
+  end
```

Comment:
Could you load default dishes from a 'default' menu potentially? What would happen here if I wanted to add my own dishes into @dishes upon initialization?